hook.php

```diff
@@ -2315,4 +2315,135 @@ function plugin_fusioninventory_getDatabaseRelations() {
    return array();
 }
 
+
+/**
+ * post_show_tab hook
+ *
+ * @param array $params Hook parameters
+ *
+ * @return void
+ */
+function postShowtab($params) {
+       switch($params['options']['itemtype']) {
+          case 'Computer':
+             if ($params['options']['tabnum'] == 1) {
+                $pfInventoryComputerComputer = new PluginFusioninventoryInventoryComputerComputer();
+                $pfComputerOperatingSystem = new PluginFusioninventoryComputerOperatingSystem();
+                $a_computerextend = current($pfInventoryComputerComputer->find(
+                                              "`computers_id`='".$params['item']->getID()."'",
+                                              "", 1));
+                if (empty($a_computerextend)) {
+                   return;
+                }
+
+                echo '<table class="tab_cadre_fixe tab_glpi" width="100%">';
+
+                if ($a_computerextend['plugin_fusioninventory_computeroperatingsystems_id'] > 0) {
+                   $pfComputerOperatingSystem->getFromDB($a_computerextend['plugin_fusioninventory_computeroperatingsystems_id']);
+                   echo '<tr>';
+                   echo '<th colspan="2">'.__('FusionInventory operating system', 'fusioninventory').'</th>';
+                   echo '</tr>';
+
+                   echo "<tr class='tab_bg_1'>";
+                   echo "<td>".__('Architecture', 'fusioninventory')."</td>";
+                   echo "<td >";
+
+                   $osArch = new PluginFusioninventoryComputerArch();
+                   if ($osArch->getFromDB($pfComputerOperatingSystem->fields['plugin_fusioninventory_computerarches_id'])) {
+                      echo $osArch->fields['name'];
+                   } else {
+                      echo '-';
+                   }
+
+                   echo "</td>";
+                   echo "</tr>";
+
+                   echo "<tr class='tab_bg_1'>";
+                   echo "<td>".__('Operating system')."</td>";
+                   echo "<td>";
+
+                   $os = new OperatingSystem();
+                   if ($os->getFromDB($pfComputerOperatingSystem->fields['operatingsystems_id'])) {
+                      echo $os->fields['name'];
+                   } else {
+                      echo '-';
+                   }
```

Comment:
Perhaps replace these lines by Dropdown::getDropdownName(), it will simplify code and reduce it.
So same for other fields in this file